codexctl/sync.py

```
@@ -43,11 +52,16 @@ def __POST(self, endpoint, data={}, fileUpload=False):
             return None
 
     def __get_documents_recursive(
-        self, folderId="", currentLocation="", currentDocuments=[]
+        self,
+        folderId: str = "",
+        currentLocation: str = "",
+        currentDocuments: list[dict[str, Any]] = [],
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)